source/_integrations/reolink.markdown

```diff
@@ -34,20 +34,24 @@ The snapshots stream provides a sequence of image snapshots giving very low late
 
 The following models have been tested and confirmed to work:
 
-- RLN8-410 NVR
+- E1 Zoom
+- E1 Outdoor
+- RLC-410W
+- RLC-420-5MP
+- RLC-511W
+- RLC-520
```

Comment:
```suggestion
- RLC-520
- RLC-522
```

Reply:
RLC-522 was also reported to be working here: https://github.com/home-assistant/core/issues/85420